external-builds/pytorch/README.md

```diff
@@ -24,19 +24,36 @@ patches locally until they can be upstreamed. See the `patches` directory.
 It is highly recommended to use a virtual environment unless if in a throw-away
 container/CI environment.
 
+On Linux:
+
 ```bash
 python -m venv .venv
 source .venv/bin/activate
 ```
 
+On Windows:
+
+```bash
+python -m venv .venv
+.venv\Scripts\activate.bat
+```
+
 ### Step 1: Preparing sources
 
+On Linux:
+
 ```bash
 # Checks out the most recent stable release branch of PyTorch, hipifies and
 # applies patches.
 ./ptbuild.py checkout
 ```
 
+On Windows:
+
+```bash
+python ptbuild.py checkout
+```
```

Comment:
At the moment, both of these can actually be `python pytorch_torch_repo.py checkout` instead (I have that change in https://github.com/ROCm/TheRock/pull/659). The underlying code is the same, but `ptbuild.py` uses different source and patches directories by default.

@lamikr having overlapping scripts and directories is going to cause more confusion until we land https://github.com/ROCm/TheRock/pull/620 or a similar change.

Reply:
adressed in https://github.com/ROCm/TheRock/pull/663/commits/f46e27815806cb1e0189054995ba59d6c28c8873